build_locally.sh

```diff
@@ -18,11 +28,22 @@ gem install jekyll
 gem install bundler -v 1.17.3
 bundle install
 
-# Build website using similar script as Travis, but skip htmlproofer because it fails locally:
-bundle exec jekyll build --strict_front_matter
+# Optionally test website using same script as Travis.
+# When you are developing locally, you may or may not want to run the full set of tests that Travis does
+if have_travis_arg "$@"; then
+  ./.travis.sh
```

Comment:
Adding the option for the test is great. However the full .travis.sh script will always fail as descried in the comment. It may be possible to achieve the best of both worlds by adding more flags to either this script and/or .travis.sh to have additional customization on what of the two tests to run, to keep this first test usable but also prevent multiple instances of the same code. however as it is just one line of code being duplicated it would just be easier and probably just as effective to just run the command "bundle exec jekyll build --strict_front_matter" instead of ./.travis.sh

Reply:
Travis run fine in the continuous integration server, so it does not need tweaking IMHO. your request is scope creep, but you're welcome to add those features in a later PR if you want.